### PR TITLE
Add new CLI tool: flare

### PR DESCRIPTION
**Pull request by dapplion**
**Motivation**

To test the merge and other forks is useful to perform non-standard actions. One of this is massive slashings. I triggered a bug during Altair testing in Prysm when sending 1000 ProposerSlashings at once. However the scripts I wrote where not high quality and eventually got lost.

`flare` is a misc tool to include multiple useful actions and resources that are useful to developers, testers and researchers.

------

**FINAL NAME**: flare

> Flare is a sudden brief burst of bright flame or light.
> In the wrong hands, can lead people astray.
> Use with care.

------

**Description**

Add new CLI separate form production focused lodestar-cli. Has two commands:

```
$ ../../node_modules/.bin/ts-node --esm src/index.ts 
Beacon chain debugging tool.
  * by ChainSafe Systems, 2018-2022

Commands:
   self-slash-proposer  Self slash validators of a provided mnemonic with ProposerSlashing
   self-slash-attester  Self slash validators of a provided mnemonic with AttesterSlashing
```

- `self-slash-proposer`: From interopIndexes or a mnemonic slashes an index range with `ProposerSlashing`. Items:
  - slot: customizable by CLI arg.
  - header roots are fixed to garbage values: 0xaaaa.., 0xbbbb..
- `self-slash-attester`: From interopIndexes or a mnemonic slashes an index range with `AttesterSlashing`: Items:
  - slot: customizable by CLI arg.
  - attestingIndices: Can specify how many validators to have in each `AttesterSlashing`.
  - index: fixed to 0
  - source: garbage values: 0xaaaa..:0
  - target: garbage values: 0xbbbb..:0

### CODE REVIEW

**Comment by wemeetagain** [comment]
:fire: `flare` :fire:

**Comment by parithosh** [comment]
Thank you so much for building this! We will definitely be using this on MSF6 or on one of the shadow forks to ensure that there are no slashing related regressions. 

Would it be possible to add a simple build + usage guide as well? 

Thank you!

File: packages/flare/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+Beacon chain multi-purpose and debugging tool.
+
+> Flare is a sudden brief burst of bright flame or light.
+> In the wrong hands, can lead people astray.
+> Use with care.
+
+## License
+
+Apache-2.0 [ChainSafe Systems](https://chainsafe.io)
```
Comment (twoeths) [inline packages/flare/README.md:19]: maybe good to add some sample commands here (could be in a separate PR)

**Review by twoeths — APPROVED** [review]

**Review by g11tech — APPROVED** [review]
